src/components/GoogleMap/MapHandler.tsx

```diff
@@ -0,0 +1,22 @@
+import { useMap } from "@vis.gl/react-google-maps";
+import React, { useEffect } from "react";
+
+interface Props {
+    place: google.maps.places.PlaceResult | null;
+}
+
+export const MapHandler = ({ place }: Props) => {
+    const map = useMap();
+
+    useEffect(() => {
+        if (!map || !place) return;
+
+        if (place.geometry?.viewport) {
+            map.fitBounds(place.geometry?.viewport);
+        }
+    }, [map, place]);
```

Comment:
тут даже возможно так 
```suggestion
    }, [map, place.geometry?.viewport]);
```

@AleksandrRazhev @KseniaRazheva как думаете?

Reply:
Так не работает зависимость начинает сыпать ошибками.